Support Arrow Left-Rigth key controls

mu-hun opened the pull request:
Left : `currentIndex - 1 + Sections.length) % Sections.length`
Right : `currentIndex + 1) % Sections.length`



Wheel 조작도 맨 끝에서 반대편으로 가는 경험을 넣으면 좋을 것 같네요.

mu-hun committed:
Support Arrow Left-Rigth key controls

Wheel 조작도 맨 끝에서 반대편으로 가는 경험을 넣으면 좋을 것 같네요.

RanolP commented on src/app/index.tsx:
```diff
@@ -87,6 +87,26 @@ const App: FC = () => {
+
+
+    if (event.key === 'ArrowLeft') {
+      updateSectionIndex((currentSectionIndex - 1 + Sections.length) % Sections.length);
```
현재 페이지 구현방식의 한계로 인해 끝 페이지에서 첫 페이지로 갈 때, 오른쪽으로 넘어가는 게 아니라 왼쪽으로 아주 빨리 넘어가는 형상을 띄게 되어 마지막 페이지와 첫 페이지가 이어지는 건 어렵겠습니다.

RanolP commented on src/app/index.tsx:
```diff
@@ -87,6 +87,26 @@ const App: FC = () => {
+    }
+
+    if (event.key === 'ArrowRight') {
+      updateSectionIndex((currentSectionIndex + 1) % Sections.length);
```
현재 페이지 구현방식의 한계로 인해 끝 페이지에서 첫 페이지로 갈 때, 오른쪽으로 넘어가는 게 아니라 왼쪽으로 아주 빨리 넘어가는 형상을 띄게 되어 마지막 페이지와 첫 페이지가 이어지는 건 어렵겠습니다.

RanolP reviewed (changes requested):
위에 적은 코멘트들에 맞춰 고쳐주세요.

ps. 늦게 리뷰하게 됐네요 ㅜㅜ

mu-hun committed:
Remove carousel style control

@RanolP : 현재 페이지 구현방식의 한계로 인해 첫 페이지에서 마지막 페이지로 갈 때,
오른쪽으로 아주 빨리 넘어가는 형상을 띄게 되어 첫 페이지가 마지막 페이지와 이어지는 건 어렵겠습니다.

RanolP commented on src/app/index.tsx:
```diff
@@ -92,13 +92,13 @@ const App: FC = () => {
-    if (event.key === 'ArrowLeft') {
-      updateSectionIndex((currentSectionIndex - 1 + Sections.length) % Sections.length);
+    if (event.key === 'ArrowLeft' && currentSectionIndex !== 0) {
+      updateSectionIndex(currentSectionIndex - 1 + Sections.length);
```
코드가 완전히 수정되지 않은 것 같아요

RanolP reviewed (changes requested):
저거 하나만 고쳐주시면 머지 가능할 것 같아요
